post_office/signals.py

```diff
@@ -0,0 +1,13 @@
+
+from django.db.models.signals import post_init
+from django.dispatch import receiver
+
+from .models import Email
+from .fields import CommaSeparatedEmailField
+
+
+@receiver(post_init, sender=Email)
+def on_email_post_init(sender, instance, **kwargs):
```

Comment:
Can we set the value attribute in `CommaSeparatedEmailField`'s `__init__` method? It seems more intuitive that way.